src/Run/CommandLine.cs

```diff
@@ -1813,7 +1825,7 @@ private static string GetEntryAssemblyName()
         private bool _noDashOnParameterSets = true;
         private bool _noSpaceOnQualifierValues;
         private string[] _parameterSetsWhereQualifiersMustBeFirst;
-        private bool _qualifiersUseOnlyDash = true;
+        private bool _qualifiersUseOnlyDash = false;
```

Comment:
Why did you want to change this default? I changed it to be true a while ago https://github.com/dotnet/buildtools/commit/c7c123ccae8390b117516b819abe09bb39fef1d1.